### ISSUE TITLE
1.06 to 1.2 artifact update script

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-model-validation-library#22]
Josef's model validation work has found a number of model inconsistencies. We need a script to update current artifacts from 1.06 to 1.1.

We should create a subdirectory in the scripts directory that contains a list of these artifact update scripts for all such updates, e.g., artifact-updates/cedar-artifact-update-1.06-to-1.1.py

A comment at the beginning of each script can list the updates it applies.

The model version task (https://github.com/metadatacenter/cedar-model-validation-library/issues/16) should be done before this script is applied and the script can update the model version accordingly.

Updates are:

(1) Null @id fields. See: https://github.com/metadatacenter/cedar-template-editor/issues/623

(2) Redundant JSON Schema in static fields. See:  https://github.com/metadatacenter/cedar-template-editor/issues/625

(3) Remove specific type constraints from numeric and date fields. See: https://github.com/metadatacenter/cedar-template-editor/issues/624

(4) Missing required field. See: https://github.com/metadatacenter/cedar-template-editor/issues/608

(5) Remove dateType field from _ui object. See: https://github.com/metadatacenter/cedar-model-validation-library/issues/3

(6) Invalid "@type": "@id" field in template field schema. See: https://github.com/metadatacenter/cedar-model-validation-library/issues/10

(7) Remove empty required fields from template element schema. See: https://github.com/metadatacenter/cedar-template-editor/issues/603

(8) Add @id fields to element instances. See: https://github.com/metadatacenter/cedar-model-validation-library/issues/21

**Comment by martinjoconnor** [comment]
Related task: #7

**Comment by martinjoconnor** [comment]
Also take a look at instances that may have not been correctly typed following the type specified in the the associated template: https://github.com/metadatacenter/cedar-template-editor/issues/602

Might be tricky so create a new task and punt if it can't be done in a reasonable time.

**Comment by martinjoconnor** [comment]
See also #32
